pkg/common/agentpathtemplate/template.go

```diff
@@ -38,12 +49,21 @@ func (t *Template) Execute(args interface{}) (string, error) {
 	if err := t.tmpl.Execute(buf, args); err != nil {
 		return "", err
 	}
-	return ensureLeadingSlash(buf.String()), nil
-}
+	path := buf.String()
 
-func ensureLeadingSlash(s string) string {
-	if len(s) > 0 && s[0] != '/' {
-		s = "/" + s
+	// Deprecated: remove in SPIRE 1.3
```

Comment:
The changelog serves as the official user-facing location for deprecations. We don't usually put a schedule on when we remove deprecated items. It is generally the next minor release, but not always (either because we don't get around to it, or we want to grant additional time for deployments to migrate away).